lib/dependency-manager-adapters/bower.js

```diff
@@ -116,18 +116,29 @@ module.exports = CoreObject.extend({
       return null;
     }
   },
-  _install() {
+  _install(depSet) {
     let adapter = this;
     return rimraf(path.join(adapter.cwd, 'bower_components'))
       .then(() => {
         debug('Removed bower_components');
-        return adapter._runBowerInstall();
+        return adapter._runBowerInstall(depSet);
       });
   },
-  _runBowerInstall() {
+  _runBowerInstall(depSet) {
     let adapter = this;
     let options = this.managerOptions || [];
-    let commandParts = ['install', '--config.interactive=false'];
+    let commandParts = ['install'];
+    options.unshift('--config.interactive=false');
+
+    // buildManagerOptions overrides all default
+    if (typeof this.buildManagerOptions === 'function') {
+      options = this.buildManagerOptions(depSet);
```

Comment:
Since the options for `bower` and `npm` are different we need to indicate _somehow_ to `buildManagerOptions` which thing we want options for.

I see two options:

1. pass two arguments (so the invocation becomes `this.buildManagerOptions('bower', depSet)`)
2. Don't support `buildManagerOptions` with bower

I personally think we should go with 2) above. Bower itself is long deprecated, and ember-try is likely to remove support for it in the next major bump; it seems silly to make a "less ergonomic API" for a new feature just to support deprecated/legacy functionality.

Reply:
Okay, I will not be supporting bower.

Reply:
@rwjblue I removed bower support.